electron/api/index.ts

```diff
@@ -0,0 +1,52 @@
+import z from 'zod'
```

Comment:
This file is basically the API routes. Here there are three procedures, all in one file for simplicity.